actors/market/src/types.rs

```diff
@@ -250,3 +250,16 @@ pub struct MarketNotifyDealParams {
     pub proposal: Vec<u8>,
     pub deal_id: u64,
 }
+
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
+#[serde(transparent)]
+pub struct ProcessDealsParams {
+    pub deal_ids: Vec<DealID>,
```

Comment:
I think we should use bitfields for this Vec<DealID> field. I need to update the FIP for this.

Reply:
I updated the FIP.